src/SaveHandler/MongoDB.php

```diff
@@ -10,62 +10,60 @@
 
 namespace Zend\Session\SaveHandler;
 
-use Mongo;
-use MongoClient;
-use MongoCollection;
-use MongoDate;
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDatetime;
+use MongoDB\Client as MongoClient;
+use MongoDB\Collection as MongoCollection;
 use Zend\Session\Exception\InvalidArgumentException;
 
 /**
  * MongoDB session save handler
  */
-class MongoDB implements SaveHandlerInterface
+final class MongoDB implements SaveHandlerInterface
```

Comment:
Sure that I should remove final and private definitions? Thought that's common if only an interface is implemented. Why do you want to allow inheritance of this class in the new major version (v3)?